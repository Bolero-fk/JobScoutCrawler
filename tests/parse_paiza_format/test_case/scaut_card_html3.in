<div class="MessageCardHeader"><img class="ScoutImage" src="sample3.jpg" alt="スカウト一覧画像"></div><div class="MessageCardBody"><div class="MessageCardTitle"><div class="MessageCardTitleName">123サービス株式会社</div><div class="MessageUnReadPoint" style="display: none;"></div></div><div class="MessageCardText">テスト説明3テスト説明3テスト説明3テスト説明3</div><ul class="MessageCardStatus"><li class="MessageCardStatusItem"><i class="fas fa-yen-sign"></i>  280万 〜 </li><li class="MessageCardStatusItem"><i class="fas fa-map-marker-alt"></i>  北海道</li><li class="MessageCardStatusItemSub"><i class="fas fa-code"></i> Ruby JavaScript</li></ul><div class="MessageCardDate"><span style="margin-right: 0.25rem;">2023/01/10</span><span style="color: rgb(46, 173, 67);">残り24日</span></div></div>