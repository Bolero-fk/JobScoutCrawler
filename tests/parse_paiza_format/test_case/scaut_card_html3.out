{"company_name": "123サービス株式会社", "description": "テスト説明3テスト説明3テスト説明3テスト説明3", "min_salary": 280, "max_salary": null, "location": "北海道", "using_lang": "Ruby JavaScript", "recieve_date": "2023/01/10", "remaining_days": 24}